cmd/nonmem.go

```diff
@@ -220,6 +261,47 @@ func generateScript(fileTemplate string, l NonMemModel) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
+func writeParaFile(l NonMemModel) error {
+	contentBytes, err := generateParaFile(l)
+
+	//Something failed during generation
+	if err != nil {
+		return err
+	}
+
+	contentLines := strings.Split(string(contentBytes), "\n")
+
+	return utils.WriteLines(contentLines, path.Join(l.OutputDir, paraFileName))
+
+}
+
+func generateParaFile(l NonMemModel) ([]byte, error) {
+	nmp := nonmemParallelDirective{
+		TotalNodes:        l.Configuration.Parallel.Nodes,
+		HeadNodes:         l.Configuration.Parallel.Nodes - (l.Configuration.Parallel.Nodes - 1),
```

Comment:
this logic will just always come to 1? why not just code it as `1`?

Reply:
Yeah. I wasn't exactly sure if I was going to have some other operation when I originally wrote it, but every single pnm file I could find always just had 1, so it does just make sense to leave it at 1. Was originally also going to do the math via functions mapped to the template and realized that was stupidly overkill for arithmetic that could be done in language.